src/main/kotlin/dev/tricht/gamesense/model/ScrollingText.kt

```diff
@@ -31,8 +32,8 @@ data class ScrollingText(
     private var originalText = _text
 
     init {
-        if (_text.length >= MAX_DISPLAY_TEXT_LENGTH) {
-            text = "$_text "
+        if (_text.length > MAX_DISPLAY_TEXT_LENGTH) {
+            text = "$_text | "
```

Comment:
I'm not sure why the `|` was added here?

Reply:
That was a preference thing to show a separation of when the title or artist name had ended and was re-looping.

Reply:
I prefer no separator, I'll add a setting for it